<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZTest_SwItemWid</class>
 <widget class="QWidget" name="ZTest_SwItemWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>输出位</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="openRadio">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>开</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="closeRadio">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>关</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
